lib/generators/apple_sign_in/config/templates/config.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+AppleSignIn.configure do |config|
+  # config.apple_aud = <Your client_id in your Apple Developer account>
+  # config.apple_iss = <Should contain 'https://appleid.apple.com'>
```

Comment:
I think there are some values that are fixed and always should be the same. The user should't change them

  iss = 'https://appleid.apple.com'
  apple keys  = 'https://appleid.apple.com/auth/keys'

Reply:
But if apple change that values for any reason? we would have to make an urgent fix

Reply:
Can you have a default value and in a case like that the user could replace it?
like `apple_iss = user_value || 'https://appleid.apple.com'` The less configuration the user has to do the better and less error, and less issues to resolve.